qcengine/programs/molpro.py

```diff
@@ -177,22 +200,36 @@ def parse_output(self, outfiles: Dict[str, str], input_model: 'ResultInput') ->
         # MP2 maps
         mp2_energy_map = {
             "total energy": "mp2_total_energy",
-            "correlation energy": "mp2_correlation_energy",
+            "correlation energy": "mp2_correlation_energy"
+        }
+        mp2_dipole_map = {"Dipole moment": "mp2_dipole_moment"}
+        mp2_extras = {
             "singlet pair energy": "mp2_singlet_pair_energy",
             "triplet pair energy": "mp2_triplet_pair_energy"
         }
-        mp2_dipole_map = {"Dipole moment": "mp2_dipole_moment"}
-        mp2_extras = {}
 
         # CCSD maps
         ccsd_energy_map = {
             "total energy": "ccsd_total_energy",
-            "correlation energy": "ccsd_correlation_energy",
+            "correlation energy": "ccsd_correlation_energy"
+        }
+        ccsd_dipole_map = {"Dipole moment": "ccsd_dipole_moment"}
+        ccsd_extras = {
             "singlet pair energy": "ccsd_singlet_pair_energy",
             "triplet pair energy": "ccsd_triplet_pair_energy"
         }
-        ccsd_dipole_map = {"Dipole moment": "ccsd_dipole_moment"}
-        ccsd_extras = {}
+
+        # CCSD(T) maps
+        # TODO There are two instances of "correlation energy" in the xml. One for CCSD and one for CCSD(T)
+        #      Will need a bit more logic to separate between the two
+        ccsd_prt_pr_energy_map = {
+            "total energy": "ccsd_prt_pr_total_energy",
+            "correlation energy": "ccsd_prt_pr_correlation_energy"
+        }
+        ccsd_prt_pr_dipole_map = {"Dipole moment": "ccsd_prt_pr_dipole_moment"}
```

Comment:
are any dipoles already in atomic units?

Reply:
The dipole is already in atomic units.